Formula/pcb2gcode.rb

```diff
@@ -21,15 +21,59 @@ class Pcb2gcode < Formula
   end
 
   depends_on "pkg-config" => :build
-  depends_on "boost"
   depends_on "gtkmm"
   depends_on "gerbv"
 
+  resource "boost" do
+    url "https://dl.bintray.com/boostorg/release/1.66.0/source/boost_1_66_0.tar.bz2"
+    sha256 "5721818253e6a0989583192f96782c4a98eb6204965316df9f5ad75819225ca9"
+  end
+
   def install
+    resource("boost").stage do
```

Comment:
Add a comment linking to the bug report, and explaining that this can be replaced when the bug is fixed.

Reply:
Done. https://github.com/Homebrew/homebrew-core/pull/30926/files#diff-104bdf5db5756d516c4cc0e80e55e48aR27